CourseGrab/ViewControllers/Search/SearchTableViewCell.swift

```diff
@@ -0,0 +1,136 @@
+//
+//  SearchTableViewCell.swift
+//  CourseGrab
+//
+//  Created by Reade Plunkett on 2/4/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableViewCell: UITableViewCell {
+
+    private let arrowImageView = UIImageView()
+    private let container = UIView()
+    private let titleLabel = UILabel()
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        // Setup appearance
+
+        selectionStyle = .none
+
+        //arrowImageView.image = UIImage(named: "logo_AppDev_black")
+        arrowImageView.backgroundColor = .black
+        arrowImageView.contentMode = .scaleAspectFit
+        container.addSubview(arrowImageView)
+
+        container.clipsToBounds = false
+        container.layer.cornerRadius = 5
+        container.layer.shadowColor = UIColor.black.cgColor
+        container.layer.shadowOpacity = 0.3
+        container.layer.shadowRadius = 2
+        container.layer.shadowOffset = .zero
+        container.backgroundColor = .white
+        contentView.addSubview(container)
+
+        titleLabel.font = ._16Semibold
+        titleLabel.numberOfLines = 0
+        container.addSubview(titleLabel)
+
+        // Setup constraints
+
+        arrowImageView.snp.makeConstraints { make in
+            make.size.equalTo(16)
+            make.centerY.equalTo(titleLabel.snp.centerY)
```

Comment:
you can just do `make.centerY.equalTo(titleLabel)`. SnapKit will assume you are referring to the same anchor if you just pass in the view